h2o-r/h2o-package/R/automl.R

```diff
@@ -100,16 +98,26 @@ h2o.automl <- function(x, y, training_frame,
                        exclude_algos = NULL,
                        include_algos = NULL,
                        modeling_plan = NULL,
+                       preprocessing = NULL,
                        exploitation_ratio = 0.0,
                        monotone_constraints = NULL,
-                       algo_parameters = NULL,
                        keep_cross_validation_predictions = FALSE,
                        keep_cross_validation_models = FALSE,
                        keep_cross_validation_fold_assignment = FALSE,
                        sort_metric = c("AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC", "AUCPR", "mean_per_class_error"),
                        export_checkpoints_dir = NULL,
-                       verbosity = "warn")
+                       verbosity = "warn",
+                       ...)
```

Comment:
Adding the `...` without documenting above causes the following warning in CRAN check.  I will push a fix. 

```
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'h2o.automl'
  ‘...’
```

Reply:
Added here: https://github.com/h2oai/h2o-3/pull/4927/commits/87c743d8c040ba45f661105ec241d46b26cbda7f